packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -57,14 +57,14 @@ export const l2LptGatewayAddresses: { [chainId: number]: string } = {
 }
 
 // Default L2 Chain to use for a certain chainId
-export const chainIdToDefaultL2ChainId: { [chainId: number]: number } = {
-  [ChainId.Mainnet]: ChainId.ArbitrumOne,
-  [ChainId.ArbitrumOne]: ChainId.ArbitrumOne,
-  [ChainId.Rinkeby]: ChainId.ArbitrumRinkeby,
-  [ChainId.ArbitrumRinkeby]: ChainId.ArbitrumRinkeby,
-  [ChainId.Goerli]: ChainId.ArbitrumGoerli,
-  [ChainId.ArbitrumGoerli]: ChainId.ArbitrumGoerli,
-  [ChainId.ArbitrumNova]: ChainId.ArbitrumNova
+export const chainIdToDefaultL2ChainId: { [chainId: number]: ChainId[] } = {
+  [ChainId.ArbitrumGoerli]: [ChainId.ArbitrumGoerli],
+  [ChainId.ArbitrumNova]: [ChainId.ArbitrumNova],
+  [ChainId.ArbitrumOne]: [ChainId.ArbitrumOne],
+  [ChainId.ArbitrumRinkeby]: [ChainId.ArbitrumRinkeby],
+  [ChainId.Goerli]: [ChainId.ArbitrumGoerli],
+  [ChainId.Mainnet]: [ChainId.ArbitrumOne, ChainId.ArbitrumNova],
```

Comment:
Now, the default network will be the first element in the array. We can be more explicit, and have a different field for the default network if we prefer:
```
const ... = {
  [ChainId.Mainnet]: {
    defaultNetwork: ChainId.ArbitrumOne,
    networks: [ChainId.ArbitrumOne, ChainId.ArbitrumNova]
  }
}
```

Reply:
I don't think this is necessary, as it's only for Mainnet